src/SFA.DAS.EmployerIncentives.Domain/ValueObjects/CollectionPeriod.cs

```diff
@@ -30,12 +31,18 @@ public CollectionPeriod(byte periodNumber, short academicYear)
         public DateTime CensusDate { get; }
         public short AcademicYear { get; }
         public bool Active { get; private set; }
+        public bool PeriodEndInProgress { get; private set; }
 
         public void SetActive(bool active)
         {
             Active = active;
         }
 
+        public void SetPeriodEndInProgress(bool periodEndInProgress)
```

Comment:
This method should be on the CollectionCalendar object since is should set to null any other periods that have the in progress set to true.  I assume only one can be running at a time.